### PR TITLE
Gh176/relperm pyscal

### PR DESCRIPTION
**Pull request by kadijahassanali**
The olio.RelPerm.text_to_relperm_dict() function now accepts a string of relative permeability data or a text file's name as arguments. This was done to improve inter-operability with the pyscal repository.

### CODE REVIEW

**Review by andy-beer — APPROVED** [review]

**Comment by kadijahassanali** [comment]
Hi @connortann, 

Would you merge this PR to the master branch please? I don't have the permission to do a merge #tragic 😁.

**Comment by connortann** [comment]
Sure - I see Andy has already reviewed. And let's sort out your permissions so you are a full maintainer, on the grounds that we generally try to get a Code Review from another maintainer before merging

**Comment by kadijahassanali** [comment]
Thanks @connortann!

**Pull request merged**
